packages/editor-ui/src/composables/useToast.ts

```diff
@@ -130,13 +131,36 @@ export function useToast() {
 			error_title: title,
 			error_description: message,
 			error_message: error.message,
+			caused_by_credential: causedByCredential(error.message),
 			workflow_id: workflowsStore.workflowId,
 		});
 	}
 
+	function showAlert(config: ElNotificationOptions): ElNotificationComponent {
+		return Notification(config);
+	}
+
+	function causedByCredential(message: string | undefined) {
+		if (!message) return false;
+
+		return message.includes('Credentials for') && message.includes('are not set');
```

Comment:
This won't work when using a non-English locale.

Reply:
Yes, but this would be a more significant change. There are multiple problems with this: why the logic is inside `showError` and/or it should receive error translation keys and not the actual text.